app/uk/gov/hmrc/crsfatcaregistration/controllers/SubscriptionController.scala

```diff
@@ -100,16 +100,18 @@ class SubscriptionController @Inject() (
         logDownStreamError(httpResponse.body)
         ServiceUnavailable(httpResponse.body)
 
-      case CONFLICT =>
+      case UNPROCESSABLE_ENTITY =>
         logDownStreamError(httpResponse.body)
-        Conflict(httpResponse.body)
+        UnprocessableEntity(httpResponse.body)
 
       case _ =>
         logDownStreamError(httpResponse.body)
         InternalServerError(httpResponse.body)
 
     }
 
+  def is2xx(status: Int): Boolean = status >= 200 && status < 300
```

Comment:
You don't need this method - it should already be available by Http library, you just need an import